docs/Python-for-Formula-Authors.md

```diff
@@ -16,11 +16,11 @@ Applications should unconditionally bundle all their Python-language dependencie
 
 ### Python declarations for applications
 
-Formulae for apps that require Python 3 **should** declare an unconditional dependency on `"python@3.x"`. These apps **must** work with the current Homebrew Python 3.x formula.
+Formulae for apps that require Python 3 **must** declare an unconditional dependency on `"python@3.x"`. These apps **must** work with the current Homebrew Python 3.x formula.
```

Comment:
Should we use the same naming convention as other doc (i.e. `3.y`)? Or perhaps other way around and change other doc to `3.x`?

Reply:
Agreed consistency would be nice, either way.

Reply:
Done